eq-author/src/tests/utils/createMockQuestionnaire.js

```diff
@@ -2,7 +2,7 @@ import { times } from "lodash";
 
 const DEFAULT_ANSWER_COUNT = 0;
 const DEFAULT_PAGE_COUNT = 1;
-const DEFAULT_FOLDER_COUNT = 1;
+const DEFAULT_FOLDER_COUNT = 2;
```

Comment:
```suggestion
const DEFAULT_FOLDER_COUNT = 1;
```

Could we keep this as 1 by default? If other tests only need one folder there's no need to inflate all the objects

Can specify `folderCount` when you call `buildQuestionnaire` if you want to override

Reply:
thanks Tom - that was a bit silly of me! sorted now